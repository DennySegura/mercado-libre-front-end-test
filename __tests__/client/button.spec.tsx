import 'jest';
import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Button from '../../src/app/components/button';

describe('Test suites for react components', () => {
  it('It should render <Button />', () => {
    const _button = renderer.create(<Button text={'Test'}/>);
    let button_tree = _button.toJSON();
    expect(button_tree).toMatchSnapshot();
  });
});
