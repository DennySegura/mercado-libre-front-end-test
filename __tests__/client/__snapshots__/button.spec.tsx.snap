// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suites for react components It should render <Button /> 1`] = `
<button
  className="button--container bg-dodger-blue"
>
  <span
    className="subheading"
  >
    Test
  </span>
</button>
`;
